src/infrastructure/validation/logger/ConsoleLogger.java

```diff
@@ -110,17 +121,47 @@ synchronized public void log(ModuleID moduleIdentifier, LogLevel level, String m
 		LocalDateTime now = LocalDateTime.now();
 		String formatDateTime = now.format(timeStampFormat);
 
-		String logTimeStamp = "["+formatDateTime+"]";
+		String logTimeStamp = "";
 
-		String logModulePart = "["+moduleIdentifier.toString()+"]";
+		String logModulePart = "";
 		
-		String logLevelPart = "["+level.toString()+"]";
+		String logLevelPart = "";
+		
+		if(!inTestMode) {
+
+			logTimeStamp = "["+formatDateTime+"]";
+			
+			logModulePart = "["+moduleIdentifier.toString()+"]";
+			
+			logLevelPart = "["+level.toString()+"]";
+	
+		} else {
+			
+			/**
+			 *  logger is in testMode
+			 *  as per infrastructure lead and test harness requirement,
+			 *  disable logs from all other modules
+			 */			
+			switch(moduleIdentifier) {
+			case INFRASTRUCTURE:
+				break;
```

Comment:
This can't be used. Content is under infra and they are using this identifier. From test harness, an empty string is being passed to logger for all parameters except message

Reply:
Checking for empty string will work

Reply:
>From test harness, an empty string is being passed to logger for all parameters except message

that does not work. that shudnt even compile because the interface parameters are: loglevel enum, moduleID enum, message string

if you want a way for it to work, then we could add new enums: TEST, and CONTENT
and allow only TEST when testMode is truthy

Reply:
check my latest commit
see it that suffices

Reply:
It suffices